### PR TITLE
bump build-number, add constrain for chardet

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,11 +26,12 @@ requirements:
...
     - certifi >=2017.4.17
-    - charset-normalizer >=2.0.0,<3
+    - charset-normalizer >=2,<3
     - idna >=2.5,<4
     - urllib3 >=1.21.1,<1.27
-    # replaced by charset normalizer and is optional as of python 3.6+: https://github.com/psf/requests/pull/5797
-    # - chardet >=3.0.2,<5
+  # https://github.com/psf/requests/blob/da9996fe4dc63356e9467d0a5e10df3d89a8528e/requests/__init__.py#L103-L115
+  run_constrained:
+    - chardet >=3.0.2,<6
```
Comment (mcg1969) [inline recipe/meta.yaml:34]: https://github.com/psf/requests/blob/da9996fe4dc63356e9467d0a5e10df3d89a8528e/requests/__init__.py#L79

This suggests it should be `3.0.2,<5` should it not?

Comment (katietz) [reply recipe/meta.yaml:34]: no, <6 is in setup.py mentioned

Comment (katietz) [reply recipe/meta.yaml:34]: I think we should give it some exra testing, but I don't see over versions (even on  major version bump to 5) any serious API/ABI breakage happening.

Comment (mcg1969) [reply recipe/meta.yaml:34]: I must be missing something. This is what I am seeing on that line.
```
    if chardet_version:
        major, minor, patch = chardet_version.split(".")[:3]
        major, minor, patch = int(major), int(minor), int(patch)
        # chardet_version >= 3.0.2, < 5.0.0
        assert (3, 0, 2) <= (major, minor, patch) < (5, 0, 0)
    elif charset_normalizer_version:
        major, minor, patch = charset_normalizer_version.split(".")[:3]
        major, minor, patch = int(major), int(minor), int(patch)
        # charset_normalizer >= 2.0.0 < 3.0.0
        assert (2, 0, 0) <= (major, minor, patch) < (3, 0, 0)
    else:
        raise Exception("You need either charset_normalizer or chardet installed")
```
I don't see 6.0.0 anywhere. Can you help me understand that?

Comment (mcg1969) [reply recipe/meta.yaml:34]: Found it! For whatever reason I was on the wrong tag. Sorry about that!

**Review by mcg1969 — APPROVED** [review]

**Pull request merged**